docs/rest_apis.rst

```diff
@@ -114,6 +117,9 @@ Cloud verifier (CV)
     :>json int verifier_port: Port of the verifier that is used.
     :>json int severity_level: Severity level of the agent. Might be `null`. Levels are the numeric representation of the severity labels.
     :>json string last_event_id: ID of the last revocation event. Might be `null`.
+    :>json int attestation_count: Number of quotes received from the agent which have verified successfully.
+    :>json int last_received_quote: Timestamp of the last quote received from the agent irrespective of validity. A value of 0 indicates no quotes have been received. May be `null` after upgrading from a previous Keylime version.
+    :>json int last_successful_quote: Timestamp of the last quote received from the agent which verified successfully. A value of 0 indicates no valid quotes have been received. May be `null` after upgrading from a previous Keylime version.
```

Comment:
Why not to consolidate `null` into 0?

Reply:
Yeah, in this case, he just followed what I have done originally, and my concern was exactly the migration problems, especially with different SQL backends, especially in case of migrations.

Reply:
That's right, I was just following the existing pattern. It does seem to me that there is a slight difference in semantics between "there has never been a successful attestation for this agent" (which results in `last_successful_attestation = 0`) and "there may have been a successful attestation for this agent but there is currently no information about when it occurred" (as would be the case after a keylime install is upgraded and the migration is run for the first time, resulting in  `last_successful_attestation = null`). But I'm not sure if this matters.